### PR TITLE
chore: release v1.0.0-8

### PR DESCRIPTION
**Pull request by seedy**
# Commits since last release

image

[page](https://github.com/traefik/faency/commits/master)

# Improvements

- [radio accordion component](https://github.com/traefik/faency/commit/099c10c713b2f7c3db55eda5416237f3af008823)

### CODE REVIEW

**Review by gndz07 — APPROVED** [review]
👍

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
```
Comment (seedy) [inline package.json:1]: @seedy It would be nice to add `TextArea` before releasing

**Review by matthieuh — APPROVED** [review]
LGTM

**Pull request merged**